.github/actions/test-base/action.yml

```diff
@@ -15,19 +15,36 @@ runs:
   steps:
     - name: repository checkout step
       uses: actions/checkout@v4
+
     - name: update tracking reference step
       run: git remote set-branches origin main
       shell: bash
+
     - name: shallow clone update step
       run: git fetch --depth 1
       shell: bash
+
     - name: python environment step
       uses: actions/setup-python@v4
       with:
         python-version: ${{ inputs.python-version-identifier }}
-    - name: dependencies installation step
+
+    - name: Display Python version
+      run: python -c "import sys; print(sys.version)"
+      shell: bash
+
+    - name: Install sktime and dependencies
       run: python3 -m pip install .[tests]
       shell: bash
+
+    - name: Show dependencies
+      run: python -m pip list
+      shell: bash
+
+    - name: Show available branches
+      run: git branch -a
+      shell: bash
```

Comment:
Similar doubt, can you please explain what is this for? Is it not always the name of the PR branch and `main`?

Reply:
This is useful to diagnose issues with, e.g., differential testing, or other utilities that call `git`.